layers/+spacemacs/spacemacs-defaults/keybindings.el

```diff
@@ -495,6 +495,7 @@
   "xtl" 'transpose-lines
   "xtp" 'transpose-paragraphs
   "xts" 'transpose-sentences
+  "xte" 'transpose-sexps
```

Comment:
Please put this in alphabetical order based on the keybinding. Also this needs to be added to the documentation.

Reply:
should be ok now.